Devscord.DiscordFramework/Middlewares/Contexts/MessageContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Devscord.DiscordFramework.Framework.Architecture.Middlewares;
+using Devscord.DiscordFramework.Framework.Commands.Parsing;
+
+namespace Devscord.DiscordFramework.Middlewares.Contexts
+{
+    public class MessageContext : IDiscordContext
+    {
+        public DateTime SendAt { get; private set; }
+        public bool IsBotCommandTest { get; private set; }
+
+        private readonly Func<bool> _getIsBotCommand;
+
+        public MessageContext(DateTime sendAt, Func<bool> botCommand)
+        {
+            this.SendAt = sendAt;
+            this._getIsBotCommand = botCommand;
+            this.IsBotCommandTest = this.IsBotCommand();
```

Comment:
coś tu chyba pozostało po testach, bo obecnie to nie ma sensu

Reply:
tutaj myśle że spokojnie może wchodzić wynik a nie funkcja
chyba nie ma potrzeby lazy loadingu
bo **chyba** to jest coś, co sprawdzamy w każdym przypadku

Reply:
Czyli zmienić żeby IsBotCommand było wynikiem funkcji?

Reply:
tak